internal/configuration/validator/server_test.go

```diff
@@ -1,6 +1,8 @@
 package validator
 
 import (
+	"io/ioutil"
```

Comment:
The stdlib package io/ioutil is discouraged for new code from 1.16 onwards., there should be equivalent methods in the io or os packages.